ios/RNPlayerView+PlayerListener.swift

```diff
@@ -1,6 +1,6 @@
 import BitmovinPlayer
 
-extension RNPlayerView: PlayerListener {
+extension RNPlayerView: PlayerListener, UserInterfaceListener {
```

Comment:
Could you either rename this file or even better extract the conformance to `UserInterfaceListener` to another `extension` in another file?

Right now this is inconsistent having both `PlayerListener` and `UserInterfaceListener` in the same file while it is named `+PlayerListener.swift`

Reply:
Sure, it makes sense. I'll extract it to another file then 🙂.

Reply:
Done 2ccbc49ef1d3f53762697396b4065fe693eab5c1